tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/trusts/ofstedPage.ts

```diff
@@ -1,7 +1,7 @@
 import { TableUtility } from "../tableUtility";
 
 class OfstedPage {
-
+    // #region Common stuffs
```

Comment:
If we are using regions we should standardise this across the other POMs, as the Governance page above is using different comments to separate the file

Reply:
This will be something I'll update as I go along - the ofsted page mainly just got done this time around as Sara was showing me what they were and how to use them etc